src/uwtools/resources/schema_with_metatasks.rng

```diff
@@ -336,6 +330,9 @@
         <ref name="compoundTimeString" />
       </element>
       <element name="timedep">
+        <attribute name="offset">
+          <data type="string" />
+        </attribute>
```

Comment:
Added missing `offset` attr to `timedep`

Reply:
Here, too, I don't think we should be modifying the RELAX NG schema. The original schema (identical to the one in the Rocoto git repo except that the formatting is tidied up) should be able to validate any Rocoto XML documents we produce, as well as ones we don't produce. This schema is the definition of what constitutes a valid Rocoto XML document, so I can't see why we'd change it. Please let me know if I'm missing something.